### ISSUE TITLE
Check how Exponential Loss function is implemented or used in Pytorch

### ISSUE THREAD

**Issue by ketankokane94** [amnag94/Accident-Prediction#4]
PyTorch has provided an implementation of NLL and Cross-Entropy check if there is an implementation for Exponential Loss too.

It's important as it's required for correct backpropagation

**Comment by ketankokane94** [comment]
Seems like easy enough to do

**Comment by ketankokane94** [comment]
WIll start with NLL first to check how the models behave.
If required will code up the Exponential Loss.
